spaces/domains.json

```
@@ -157,5 +157,6 @@
   "governance.curio.cards": "curiovoters.eth",
   "vote.forkenomics.com": "forkenomics.eth",
   "snapshot.comp.vote": "comp-vote.eth",
-  "gov.tusker.finance": "tuskerfinance.eth"
+  "gov.tusker.finance": "tuskerfinance.eth",
+  "vote.macaronswap.finance": "macaronswap"
```

Your space domain selector should be similar to your ENS.
```suggestion
  "vote.macaronswap.finance": "macaronswap.eth"
```